src/applications/gi/reducers/calculator.js

```diff
@@ -86,6 +87,17 @@ export default function(state = INITIAL_STATE, action) {
         [field]: convertedValue,
       };
 
+      if (field === 'EYBVetTecProgram') {
```

Comment:
can we avoid the use of EYB in variable names if it is not needed to distinguish a new variable from an old variable

Reply:
Done. I wasn't sure what would have been the best name for this.